superset-frontend/cypress-base/cypress/integration/dashboard/save.test.js

```diff
@@ -16,55 +16,118 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+import shortid from 'shortid';
 import readResponseBlob from '../../utils/readResponseBlob';
 import { WORLD_HEALTH_DASHBOARD } from './dashboard.helper';
 
 describe('Dashboard save action', () => {
-  let dashboardId;
-
   beforeEach(() => {
     cy.server();
     cy.login();
     cy.visit(WORLD_HEALTH_DASHBOARD);
+  });
 
-    cy.get('#app').then(data => {
-      const bootstrapData = JSON.parse(data[0].dataset.bootstrap);
-      const dashboard = bootstrapData.dashboard_data;
-      dashboardId = dashboard.id;
-      cy.route('POST', `/superset/copy_dash/${dashboardId}/`).as('copyRequest');
-    });
+  // it('should save as new dashboard', () => {
```

Comment:
protip: use `xit` in place of `it` to disable a test

Reply:
👍 I uncommented it. Thanks for catching!

Reply:
that was just for temporary testing purposes while this was a draft.